src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,173 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Class to test movie database.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase moviedb1;
+    private MovieDatabase moviedb2;
+    private Movie movie1;
+    /**
+     * Method to setup movie1 and movie2.
+     */
+
+    @Before
+    public void setUpMovieDatabases() {
+        String actorAffleck = "Ben Affleck";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Ben Affleck" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)